### PR TITLE
[Method References] LPS-162473 Convert method references to field references in class entends BaseConfigurationProvider

### PR DESCRIPTION
**Pull request by W2jx**
Hi @tinatian @dantewang , I keep the proected configurationAdmin field and directly add reference to it.

Detial: https://github.com/liferay-core-infra/liferay-portal/pull/1625

### CODE REVIEW

**Comment by W2jx** [comment]
Hi @catarineaguiar , Could you please help me check the test results ? Thanks!

**Comment by catarineaguiar** [comment]
@W2jx no regressions.

File: modules/apps/portal-security/portal-security-ldap-api/src/main/java/com/liferay/portal/security/ldap/configuration/BaseConfigurationProvider.java [hunk 1]
```diff
@@ -46,9 +47,7 @@ protected String getMetatypeId() {
 		return _factoryPid;
 	}
 
-	protected abstract void setConfigurationAdmin(
-		ConfigurationAdmin configurationAdmin);
-
+	@Reference
```
Comment (tinatian) [inline modules/apps/portal-security/portal-security-ldap-api/src/main/java/com/liferay/portal/security/ldap/configuration/BaseConfigurationProvider.java:50]: @W2jx The change here is weird, the current class is not a component, but you are using @Reference.

Tina.

**Comment by tinatian** [comment]
@W2jx Please see my comment and resend.

Tina

**Pull request closed without merging**